etc/rules/syslog_rules.xml

```diff
@@ -579,7 +579,7 @@
 <group name="syslog,dpkg,">
   <rule id="2900" level="0">
     <decoded_as>windows-date-format</decoded_as>
-    <regex>^\d\d\d\d-\d\d-\d\d \d\d:\d\d:\d\d \w+ </regex>
+    <regex offet="after_parent">^startup |^status |^remove |^configure |^install |^purge |^trigproc </regex>
```

Comment:
Reported this "offet" typo as a new issue in https://github.com/ossec/ossec-hids/issues/1433